style.css

```diff
@@ -0,0 +1,123 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: "Trebuchet MS", sans-serif;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+	background-color: #e6f2f4;
+}
+
+nav {
+	font-size: 1.2rem;
+	height: 75px;
+	width: 100%;
+	background-color: #a7d9f4;
+	display: flex;
+	justify-content: space-around;
+	flex-direction: row;
+	align-items: center;
+}
+
+.navbar {
+	text-decoration: none;
+	color: black;
+}
+
+
+h1, h2, .me, .info, .projects {
+	text-align: center;
+	padding: 5px;
+}
+
+.personal-photo {
+	height: 300px;
+	width: 250px;
```

Comment:
add styling to make sure you don't end up like this
![image](https://github.com/CMeier556/PortfolioProject/assets/25844355/9f0b87ac-48c1-4bf4-8f0b-6fc84d0fd7ee)